pkg/bugreport/controlplane.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/openservicemesh/osm/pkg/constants"
 )
 
-func (c *Config) collectControlPlaneLogs() error {
+func (c *Config) collectControlPlaneLogs() {
```

Comment:
The main drawback to not returning an error here anymore is that the overall exit status won't reflect if an error was hit here. I wonder if accumulating errors in something like a `multierror` would let us both continue when we hit errors while exposing whether or not errors were hit.

Reply:
This is consistent with log collection for other components - meshes, app namespaces, app pods etc.
The completion success/failure messages indicate when a step succeeds/fails.

Reply:
Agreed; I tend to hesitate when making a change that removes information. A multierror or similar implementation would be what I prefer.

Reply:
This does not remove error info, it correctly gets printed within the function, which was previously not the case due to the early return on error semantics.

Reply:
Ah I see now; that's what completionFailure does. LGTM